common/components/Header/components/TabsOptions.jsx

```diff
@@ -23,7 +23,8 @@ const tabs = [
         link: 'view-wallet'
     },
     {
-        name: 'Help'
+        name: 'Help',
```

Comment:
Probably a better convention than this - maybe react-router even has a way of listing all routes. 

Can fix in a separate PR.